api/app/analysis/dist_alerts/analysis.py

```diff
@@ -125,13 +127,21 @@ async def do_analytics(file_path):
             """
             CREATE OR REPLACE SECRET secret (
                 TYPE s3,
-                PROVIDER credential_chain
+                PROVIDER credential_chain,
+                CHAIN config
             );
         """
         )
 
-        # Send query to DuckDB and convert to return format
+        # PZB-271 just use DuckDB requester pays when this PR gets released: https://github.com/duckdb/duckdb/pull/18258
+        # For now, we need to just download the file temporarily
```

Comment:
Thanks for the note. 💯 
You could also mark it with a TODO if you want to be extra annoying to our future selves. 😄